net/lnet/lnet_main.c

```diff
@@ -922,7 +922,7 @@ M0_BASSERT(M0_NET_LNET_PID == LUSTRE_SRV_LNET_PID);
 M0_INTERNAL int m0_net_lnet_init(void)
 {
 	m0_net_xprt_register(&m0_net_lnet_xprt);
-	m0_net_xprt_default_set(&m0_net_lnet_xprt);
+	/*m0_net_xprt_default_set(&m0_net_lnet_xprt);*/
```

Comment:
Please add comments to explain why this is deprecated.
Or we need to do:
```
#if !ENABLE_LIBFAB
        m0_net_xprt_default_set(&m0_net_lnet_xprt);
#endif
```

Reply:
We had commented this to ensure that lnet is not set as the default transport during our testing and for the custom build. 
Reverted this change to set lnet as the default transport.